config/reports/ukraineSubsidyOproep1Report.js

```diff
@@ -87,26 +86,23 @@ export default {
     const queryResponse = await batchedQuery(queryString);
     const data = queryResponse.results.bindings.map((subsidie) => {
       return {
-        subsidieURI: getSafeValue(subsidie, "subsidie"),
-        modified: getSafeValue(subsidie, "modified"),
-        status: getSafeValue(subsidie, "status"),
-        bestuurseenheid: getSafeValue(subsidie, "bestuurseenheid"),
-        kbo: getSafeValue(subsidie, "kbo"),
-        classificatie: getSafeValue(subsidie, "classificatie"),
-        contactFirstName: getSafeValue(subsidie, "contactFirstName"),
-        contactLastName: getSafeValue(subsidie, "contactLastName"),
-        contactTelephone: getSafeValue(subsidie, "contactTelephone"),
```

Comment:
Similar to the KBO field, removing the double quotes on the telephone number column leads to zeros being truncated. In addition, the column no longer stays "balanced"; in other words, without the double quotes, some phone numbers are aligned to the left (e.g., those that have spaces or contain the `/` character), and other phone numbers are aligned to the right (e.g., numbers like 04xxxxxx), which makes the column less readable than before.